genai-perf/genai_perf/parser.py

```diff
@@ -650,6 +694,14 @@ def _add_endpoint_args(parser):
         "you must specify an api via --endpoint-type.",
     )
 
+    endpoint_group.add_argument(
+        "--server-metrics-url",
+        type=str,
+        default=None,
+        required=False,
+        help="The full URL to access the server metrics endpoint. This argument is required if the metrics are available on a different machine than localhost (where GenAI-Perf is running).",
```

Comment:
This is inconsistent with url + endpoint above.
The problem is that the url + endpoint should be passed in as a single arg and we slice it internally but for now its inconsistent.
This isnt wrong but it might cause UX issues.